nextjs-app-v14/package.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
will we generate a new next 14 app to go in here later? I have no idea how trivial it is to add everything by hand later on